src/hooks/useSearchNotes.js

```diff
@@ -0,0 +1,31 @@
+import { useState } from "react";
+import { useSnackbarAlerts } from "./useSnackbarAlerts";
+
+export const useSearchNotes = () => {
```

Comment:
when we export one file from a file with the function name we tend to use default export 
```suggestion
const useSearchNotes = () => {

export deafult useSearchNotes 
```